tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -164,6 +164,16 @@ describe('Acceptance: addon-smoke-test', function() {
       });
   });
 
+  it('ember addon with linting errors', function() {
+    return copyFixtureFiles('addon/with-linting-errors')
+      .then(function() {
+        return ember(['test']);
+      })
+      .then(function(result) {
```

Comment:
The `ember` helper should reject with non-zero exit codes so this `then` is likely not being called (probably needs to change to `.catch`).  This kind of thing makes be wish for `assert.expect(1)` from QUnit...

https://github.com/ember-cli/ember-cli/blob/master/tests/helpers/ember.js#L85